tensorflow/lite/kernels/internal/reference/strided_slice.h

```diff
@@ -18,74 +18,51 @@ limitations under the License.
 #include "tensorflow/lite/kernels/internal/common.h"
 #include "tensorflow/lite/kernels/internal/strided_slice_logic.h"
 #include "tensorflow/lite/kernels/internal/types.h"
+#include <functional>
```

Comment:
This implementation is shared with TFLite Micro so we avoid using <functional>.
could you move the lambda to a separated function